readme.md

```diff
@@ -81,7 +89,12 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+
+
+# pbj = Sandwich.new(id: 1, type "Peanut-Butter & Jelly", student_id: 1)
+
+# Geraldo = Student.create(id: 1, name: Geraldo )
+
 
```

Comment:
Couple things here...

1. You cannot assign a sandwich to a student that does not exist. If you are going to reference `student_id: 1`, then that students needs to be created before that line of code runs. So in this case, `Geraldo` should be created before `pbj`.

2. When creating a new instance, you should not be assigning the `id` yourself. Rails with automatically generate the `id`.